devices/grlib-multi/test-grlib-multi.c

```diff
@@ -775,6 +775,5 @@ void runner(void)
 
```

Comment:
Please correct it also in waitpid test. We should also consider corrections in other repos - for example phoenix-rtos devices